components/stacks-network/src/event.rs

```diff
@@ -86,10 +86,11 @@ pub fn send_status_update(
         "stacks-node" => 1,
         "stacks-signers" => 2,
         "stacks-api" => 3,
-        "subnet-node" => 4,
-        "subnet-api" => 5,
-        "stacks-explorer" => subnet_services + 4,
-        "bitcoin-explorer" => subnet_services + 5,
+        "postgres" => 4,
```

Comment:
I'm not sure we want to have it here, displayed in the TUI. We didn't have it before, and it might not be relevant for common use cases. What do you think?